Dockerfile

```diff
@@ -1,4 +1,14 @@
 ARG  SERVERCORE_VERSION
 
+FROM golang:1.16-nanoserver-1809 AS build-env
+
+WORKDIR /project
+ADD . /project
+RUN cd /project
+RUN go build -o bin/wincat.exe cmd/wincat/main.go
+#RUN go build -o bin/pauseloop.exe cmd/pauseloop/main.go
```

Comment:
drop comments if they dont do anything? we dont need pauseloop for sure?

Reply:
Actually I wasn't sure if pauseloop is needed...